pkg/recutil/reconcile.go

```diff
@@ -176,3 +178,23 @@ func RoleBindingDiff(expectedObj runtime.Object, existingObj runtime.Object) Out
 
 	return operation
 }
+
```

Comment:
We could now remove the `RoleBindingDiff` func. I don't think there's any use for it; as the RBAC controller does things in a different way

Reply:
I removed it, it's no need it anymore.